src/plots/gl3d/scene.js

```diff
@@ -58,10 +58,10 @@ function render(scene) {
     }
 
     function formatter(axisName, val) {
-        if(typeof val === 'string') return val;
-
         var axis = scene.fullSceneLayout[axisName];
-        return Axes.tickText(axis, axis.c2l(val), 'hover').text;
+        var calendar = trace[axis._id.charAt(0) + 'calendar'];
+
+        return Axes.tickText(axis, axis.d2l(val, 0, calendar), 'hover').text;
```

Comment:
I don't see a mock with category data in 3D, though it does work:
```js
Plotly.newPlot(gd,[{x:[1,2,3],y:['a','b','c'],z:['2014-01-01','2016-01-01','2015-01-01'], type:'scatter3d'}])
```
I *think* this code is compatible with category data, but can you include a test for it?